docs/en/contributing/install.md

```diff
@@ -8,33 +8,28 @@ sidebar_label: Install
 
 ## Prerequisites
 
-Before setting up FAST-DNA, install Git and [yarn](https://yarnpkg.com/en/docs/install).
+Before setting up FAST-DNA, install Git and <a href="https://yarnpkg.com/en/docs/install" target="_blank">Yarn</a>.
```

Comment:
What is the purpose of changing this to be an anchor here (and below) rather than leveraging the markdown syntax?

Reply:
Linking external / 3rd party tools and resources via a new window is a best practice when writing content.

Reply:
I'm not going to block on this, but is there a source of truth we are using to determine that? This is changing the default behavior of the links on the page and setting a new default to **always** open in a new tab. Currently if a user wants to open in a new tab there are a couple ways they can do that - 1) context menu, and 2) keyboard shortcut. 

Letting the user choose how to manage their browsing workflow seems to be the ideal here and in most situations. There may be exceptions when you absolutely want to pick a different default as is being done here (a user is doing work on a page, form submission where work could be lost, etc) but I'm not sure this is the same.

Reply:
I don't think this was resolved, so I'm reopening this as unresolved. That does not stop the PR from going in with enough approvals, but my response wasn't addressed so I don't think we can mark the conversation as "resolved".

Reply:
This doesn't change the defaults on the page, it only changes the link that uses HTML Anchor instead of MD.  Your points are valid.  Users still have the option to open in a new tab or new browser as you suggest.  We're not taking that from them.  However, we are attempting to keep users on our site for SEO purposes which better supports our goals.

Reply:
Here is the first resource I found https://www.bluecorona.com/blog/outbound-links-seo-best-practices/  If you still feel strongly about this topic you can raise it to the steering committee.